.github/CONTRIBUTING.md

```diff
@@ -64,6 +62,43 @@ To build containerized `moby/buildkit:local` and `moby/buildkit:local-rootless`
 make images
 ```
 
+### Run BuildKit
+
+You can launch the backend BuildKit daemon either directly or in a container:
+
+```bash
+# run the daemon directly
+$ sudo buildkitd --addr tcp://localhost:1234
+$ export BUILDKIT_HOST=tcp://localhost:1234
+$ buildctl debug info
+BuildKit: github.com/moby/buildkit v0.11.0-rc3-506-g539bab193.m 539bab193c28d3ce731e6013f471ba24848f5c41.m
+```
+
+```bash
+# run the daemon in a container
+$ docker run --rm -d --name buildkitd --privileged moby/buildkit:local
+$ export BUILDKIT_HOST=docker-container://buildkitd
+$ buildctl debug info
+BuildKit: github.com/moby/buildkit v0.11.0-rc3-623-g2ff0d2a2f.m 2ff0d2a2f53663aae917980fa27eada7950ff69c.m
+```
+
+You can also connect buildx to the BuildKit daemon using the [`remote` driver](https://docs.docker.com/build/drivers/remote/):
+
+```bash
+$ docker buildx create --driver=remote --name=dev $BUILDKIT_HOST
+$ docker buildx --builder=dev inspect
+Name:          dev
+Driver:        remote
+Last Activity: 2023-06-06 14:15:52 +0000 UTC
+
+Nodes:
+Name:      dev0
+Endpoint:  tcp://localhost:1234
+Status:    running
+Buildkit:  v0.11.0-rc3-506-g539bab193.m
+Platforms: linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/386
+```
+
```

Comment:
We could also put a full container example with buildx in here. I often share it when people ask the easiest way to test custom buildkit.

```
BUILDX_BUILDER=default docker buildx build -t moby/buildkit:dev --load .
export BUILDX_BUILDER=dev
docker buildx rm "$BUILDX_BUILDER" || true # remove previous dev builder if exists
docker buildx create --name=dev --driver-opt image=moby/buildkit:dev --bootstrap
```

Reply:
Added a new section below for how to do this.